src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -62,6 +65,12 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
   @Input()
   flag?: string;
 
+  total = 100;
+
+  warningThreshold = 0.8;
```

Comment:
Is this for CPU usage when any daemon reaches it's maximum CPU and memory usage from the threshold set, Does it warn the user?